packages/docs/stories/components/range.stories.ts

```diff
@@ -0,0 +1,453 @@
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'prefix-suffix'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range
+      help-text="Controls the volume of the current song"
+      label="Volume"
+      max="100"
+      min="0"
+      value="50"
+    >
+      <span slot="prefix">0</span>
+      <span slot="suffix">100</span>
+    </syn-range>
+
+    <p style="margin: var(--syn-spacing-x-large) 0;">This can be used to add input fields or icons.</p>
+
+    <syn-range label="Estimated Time" class="suffix-input-field" min="0" max="60" value="30">
+      <span slot="prefix">0</span>
+      <span slot="suffix">
+        <div class="suffix-input">
+          60
+          <syn-input value="30" type="number" no-spin-buttons min="0" max="60">
+            <span slot="suffix">sec</span>
+          </syn-input>
+        </div>
+      </span>
+    </syn-range>
+    
+    <style>
+      .suffix-input {
+        align-items: center;
+        display: flex;
+        gap: var(--syn-spacing-medium);
+        syn-input {
+          min-width: 0;
+        }
+      }
+    </style>
+    <script>
+      [...document.querySelectorAll('.suffix-input-field')]
+        .forEach(range => {
+          const input = range.querySelector('syn-input');
+
+          range.addEventListener('syn-input', e => {
+            const { target } = e;
+            if (target.tagName !== 'SYN-RANGE') return;
+            target.querySelector('syn-input').value = target.value;
+          });
+
+          input.addEventListener('syn-input', e => {
+            const { target } = e;
+            target.closest('syn-range').value = target.value;
+          });
+        });
+    </script>
+  `,
+};
+
+export const CustomTrackColors: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'custom-track-colors'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range
+      class="custom-track-color"
+      max="100"
+      min="0"
+      value="50"
+    ></syn-range>
+    <style>
+      .custom-track-color {
+        --track-color-active: var(--syn-color-success-600);
+      }
+    </style>
+  `,
+};
+
+export const CustomTrackOffset: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'custom-track-offset'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range
+      class="custom-track-offset"
+      max="100"
+      min="0"
+      value="25"
+    ></syn-range>
+    <style>
+      .custom-track-offset {
+        --track-active-offset: 50%;
+      }
+    </style>
+  `,
+};
+
+export const MultiKnob: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'multi-knob'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range max="100" min="0" value="30 70"></syn-range>
+  `,
+};
+
+export const MultiKnobWithRestrictedMovement = {
```

Comment:
<video src="https://github.com/user-attachments/assets/0b1bc4c6-cb1d-472d-a2ff-f5bed1dd7fd2"></video>

It behaves a bit strange when I try to move, depending on speed etc. it's not at the same position as the right knob. (Safari)

Reply:
I see what you mean, have the same problem. I will try to have a look at what this is causing. Maybe it is because the hover of the other knob interfers and slallows the event.

Reply:
This is now fixed. It was lazy programming, not the event type itself :). Also, with the latest changes it also works with keyboard navigation (which was previously unaccounted)